gravitee-am-ui/src/app/domain/settings/password-policies/domain-password-policies.component.html

```diff
@@ -42,10 +42,29 @@ <h1>Password Policies</h1>
           <div matBadge="{{ row.idpsNames ? row.idpsNames.length : 0 }}" matTooltip="{{ getTooltipText(row.id) }}"></div>
         </ng-template>
       </ngx-datatable-column>
-      <ngx-datatable-column name="Default" [flexGrow]="1" [resizeable]="false" [sortable]="false"></ngx-datatable-column>
+      <ngx-datatable-column name="Default" [flexGrow]="1" [resizeable]="false" [sortable]="false">
+        <ng-template ngx-datatable-header-template>
+          <div style="display: flex">
+            <span style="margin-right: 3px">Default</span>
+            <span
+              ><mat-icon
+                style="font-size: 18px; color: black; cursor: pointer"
+                matTooltip="Assign a policy for Identity providers not configured with a specific policy."
+              >
+                info_outline</mat-icon
+              ></span
+            >
```

Comment:
this line breaks looks wired

Reply:
This is linter work :D

Reply:
When I changed it and run `npm run lint:fix` it changed back to this formatting.

Reply:
you can put it in one line and then lint. The result will be different